Installing libffi...
  log file is /opt/rapd/share/cctbx/base_tmp/libffi_install_log
  getting package libffi-3.2.1.tar.gz...
    downloading from http://cci.lbl.gov/cctbx_dependencies : 0.9 MB
    [0%.........20%.........40%.........60%.........80%.........100%]
  installing libffi-3.2.1.tar.gz...
Installing glib...
  log file is /opt/rapd/share/cctbx/base_tmp/glib_install_log
  getting package glib-2.46.2.tar.gz...
    downloading from http://cci.lbl.gov/cctbx_dependencies : 14.1 MB
    [0%.........20%.........40%.........60%.........80%.........100%]
  installing glib-2.46.2.tar.gz...
Traceback (most recent call last):
  File "modules/cctbx_project/libtbx/auto_build/install_base_packages.py", line 1457, in <module>
    installer(args=sys.argv, log=sys.stdout)
  File "modules/cctbx_project/libtbx/auto_build/install_base_packages.py", line 207, in __init__
    self.build_dependencies(packages=packages)
  File "modules/cctbx_project/libtbx/auto_build/install_base_packages.py", line 637, in build_dependencies
    getattr(self, 'build_%s'%i)()
  File "modules/cctbx_project/libtbx/auto_build/install_base_packages.py", line 1105, in build_glib
    log=pkg_log)
  File "modules/cctbx_project/libtbx/auto_build/install_base_packages.py", line 501, in configure_and_build
    self.call("./configure %s" % " ".join(list(config_args)), log=log)
  File "modules/cctbx_project/libtbx/auto_build/install_base_packages.py", line 302, in call
    return call(args, log=log, verbose=self.verbose, **kwargs)
  File "/opt/rapd/share/cctbx/modules/cctbx_project/libtbx/auto_build/installer_utils.py", line 81, in call
    raise RuntimeError("Call to '%s' failed with exit code %d" % (args, rc))
RuntimeError: Call to './configure --prefix="/opt/rapd/share/cctbx/base"' failed with exit code 1
Process failed with return code 1
Could not install cctbx


